contrib.php

```diff
@@ -15,6 +15,12 @@
 // multiuser permission check
 permission_or_die(PERM_WRITE);
 
+function arrayEach(&$arr) {
```

Comment:
while this may work, I wouldn't trust a function that I can't grasp by looking at it. This seems like a maintenance nightmare.

Reply:
drop this function now?

Reply:
removed